### PR TITLE
Update BMLacq680

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
The images of BMLacq680 are not complete. The scan images for folio 6v-7r (after scan page 8) as well as for folio 179v-180r (after scan 180). It can be discerned from the folio marks (bottom left) and from the text, that is not fitting.

In encoding the quire structure and the description of hands, I differed a lot from P. Marrassini. As well the last textpart (ms_i3 and ms_i4) are encoded independently contrary to Marrassinis account (See also Issue  #2367). Also I treated the index on f. 4v, I as an addition because it is by another hand as most of ms_i1 but not as the incipit of ms_i1, as Marrassini did.

### CODE REVIEW

File: FlorenceBML/BMLacq680/BMLacq680.xml [hunk 1]
```diff
@@ -111,15 +111,14 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                                    ዘወጽአ፡ እምብሉይ፡ ወሐዲስ።</explicit>
+                                    ፈቃዱን፡ በሠሩለት፡ ከሰይጣን፡ ጋራ፡ ዘለዓለም፡ ሲሣቀዮ፡ ይኖራሉ። ከዚህ፡ ሃይማኖት፡ አያናውጸን፡ አሜን። ዝመጽሐፍ፡ 
+                                    ዘ<persName ref="PRS2435Balcasa" role="owner"><roleName type="title">ደጃዝማች፡</roleName> ባልቻ፡</persName> 
+                                    ወስመ፡ ጥምቀቱ፡ <persName ref="PRS2435Balcasa" role="owner">ተክለ፡ ሚካኤል፡</persName> ዘወጽአ፡ እምብሉይ፡ ወሐዲስ። </explicit>
                             </msItem>
                         </msItem>
                         <msItem xml:id="ms_i2">
                            <locus from="29ra" to="291ra"/>
-                           <title ref="LIT2375Synaxa"/>
+                            <title ref="LIT4033Senkessar"/>
```
Comment (thea-m) [inline FlorenceBML/BMLacq680/BMLacq680.xml:121]: It's probably the "vulgate" second recension, LIT4035SenkessarDL

Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq680/BMLacq680.xml:121]: OK. I will change it to the more specific ID.

**Review by eu-genia — APPROVED** [review]

File: FlorenceBML/BMLacq680/BMLacq680.xml [hunk 2]
```diff
@@ -35,6 +36,578 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             <idno>Marrassini ms. 15</idno>
```
Comment (eu-genia) [inline FlorenceBML/BMLacq680/BMLacq680.xml:36]: I have inserted empty pages for missing images, please change `n="309"` in `facs` above

(`<idno facs="Laurenziana/BML_015/BML_015-" n="309">BML Acq. e doni 680</idno>`)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: FlorenceBML/BMLacq680/BMLacq680.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply FlorenceBML/BMLacq680/BMLacq680.xml:36]: Ok. I changed 307 for 309 in idno.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**